platform/android/os_android.cpp

```diff
@@ -335,5 +353,24 @@ OS_Android::OS_Android(GodotJavaWrapper *p_godot_java, GodotIOJavaWrapper *p_god
 	DisplayServerAndroid::register_android_driver();
 }
 
+Error OS_Android::create_process(const String &p_path, const List<String> &p_arguments, ProcessID *r_child_id) {
+	if (p_path == "/system/bin/app_process64") {
```

Comment:
not sure if it will work on 32 bit devices 😅

Reply:
> not sure if it will work on 32 bit devices 😅

I am ready to test if you release 32 bit build. :]

Reply:
When creating a new godot process it tries to read what is the current process, this is what it always return in android, so if godot trying to read this process it means godot trying to create a new Godot process so I created a new activity instead of a new process (but if another path is passed instead it should run it)

Reply:
Addressed in https://github.com/thebestnom/godot/pull/2